src/css/styles.scss

```diff
@@ -1,12 +1,8 @@
-// COLORS VARS
-:root {
-  --bcg-color: #111f28;
-  --section-bcg-color: #484e52;
-  --base-color: #ffffff;
-  --accent-color: #dda0dd;
-}
-
 $base-font: 'Open Sans', sans-serif;
+$bcg-color: var(--bcg-color);
+$section-bcg-color: var(--section-bcg-color);
+$base-color: var(--base-color);
+$accent-color: var(--accent-color);
```

Comment:
What does the use pf the `--` here represent? Are you just setting the variable `$bcg-color` to a new variable? Is this different than  ` --bcg-color: #c0dbf5` ? Could we not just save our `$bcg-color =  #c0dbf5` ? Thanks!